### PR TITLE
Fix bugs related to `go_version_file` input

### PR DESCRIPTION
**Pull request by upamune**
## WHAT

1. Added `go_version_file` to `action.yml` to avoid a warning
2. Remove the default value of `go_version` from `action.yml` and add the deafult Go version to the code.

## WHY

1. To avoid a below warning
>  Warning: Unexpected input(s) 'go_version_file', valid inputs are ['github_token', 'golangci_lint_flags', 'tool_name', 'level', 'reporter', 'filter_mode', 'fail_on_error', 'reviewdog_flags', 'workdir', 'reviewdog_version', 'golangci_lint_version', 'go_version', 'cache']
2.  If the `go_version` default value is set in `action.yml`, the `go_version_file` value is never used. This is because if both are set, the `go_version` value takes precedence. Therefore, I've modified it to use the default version if `go_version` and `go_version_file` are not set.

### CODE REVIEW

**Comment by upamune** [comment]
@shogo82148 Thank you for reviewing the #389. I actually tried to run it, but found problems. Therefore, I created this PR. Could you review it?

File: src/main.ts [hunk 1]
```diff
@@ -31,11 +31,9 @@ async function run() {
...
     const enableCache = core.getBooleanInput("cache");
 
-    if (goVersion !== "" || goVersionFile !== "") {
-      await core.group("Installing Go ...", async () => {
-        await setupGo.run(goVersion, goVersionFile);
-      });
-    }
+    await core.group("Installing Go ...", async () => {
+      await setupGo.run(goVersion, goVersionFile);
+    });
```
Comment (upamune) [inline src/main.ts:36]: When a user wants to use the default `goVersion`, the `goVersion` is passed as an empty string, so I removed this `if` block.

**Review by shogo82148 — APPROVED** [review]
LGT

**Comment by upamune** [comment]
Thank you for your quick review!  I tried the following pattern on my project and confirmed that it works correctly.

- specify only `go_version`
- specify only `go_version_file`
- specify both `go_version` and `go_version_file`
- specify nothing

**Pull request merged**